eventuate-common-jdbc/src/main/java/io/eventuate/common/jdbc/EventuateCommonJdbcOperations.java

```diff
@@ -33,23 +33,22 @@ public void insertIntoEventsTable(String eventId,
   }
 
 
-  public void insertIntoMessageTable(String messageId,
-                                      String payload,
+  public Long insertIntoMessageTable(String payload,
```

Comment:
This and perhaps elsewhere should be `long`

Reply:
ok